### PR TITLE
Example to boot a shell with Alpine's minirootfs and stage0 firmware.

### PR DESCRIPTION
**Pull request by bgogul**
Note that instead of packaging the init binary as rust binary, we have simply created a shell script with the appropriate initialization commands.

### CODE REVIEW

File: oak_minirootfs_linux_init/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+( cd stage0; cargo build --release; )
+objcopy --output-format binary stage0/target/x86_64-unknown-none/release/oak_stage0 \
+    bin/stage0.bin
+```
+
+Execute the initial RAM disk with QEMU:
+
+Note: this assumes that an appropiate
+[uncompressed Linux kernel ELF binary](/docs/development.md#extracting-vmlinux-from-your-linux-installation)
+has been copied to `bin/vmlinux`.  The linux kernel that comes with Alpine distribution [optimized for virtual systems](https://dl-cdn.alpinelinux.org/alpine/v3.17/releases/x86_64/alpine-virt-3.17.3-x86_64.iso) is suitable.
```
Comment (conradgrobler) [inline oak_minirootfs_linux_init/README.md:51]: nit: "The Linux kernel ...."

File: oak_minirootfs_linux_init/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<!-- Oak Logo Start -->
+<!-- An HTML element is intentionally used since GitHub recommends this approach to handle different images in dark/light modes. Ref: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to -->
+<!-- markdownlint-disable-next-line MD033 -->
+<h1><picture><source media="(prefers-color-scheme: dark)" srcset="../docs/oak-logo/svgs/oak-logo-negative.svg?sanitize=true"><source media="(prefers-color-scheme: light)" srcset="../docs/oak-logo/svgs/oak-logo.svg?sanitize=true"><img alt="Project Oak Logo" src="../docs/oak-logo/svgs/oak-logo.svg?sanitize=true"></picture></h1>
+<!-- Oak Logo End -->
+
+# Shell with Alpine's Mini Root Filesystem 
+
+This example shows how to boot into a shell from Oak's stage0 firmware binary. We can achieve this by simply packaging [Alpine Linux](https://www.alpinelinux.org)'s mini root filesystem as the [initial RAM disk](https://en.wikipedia.org/wiki/Initial_ramdisk) (initramfs) disk with an appropriate init script. The easiest way to execute the initial RAM disk is to use
```
Comment (conradgrobler) [inline oak_minirootfs_linux_init/README.md:9]: The format checks are failing. Running `xtask format` should fix it.

**Review by conradgrobler — APPROVED** [review]
Thanks!

**Review by tiziano88 — APPROVED** [review]
Neat! Thanks Gogul

File: oak_minirootfs_linux_init/README.md (hunk 2 above)
Comment (bgogul) [reply oak_minirootfs_linux_init/README.md]: done.

**Pull request merged**